test/b2gedmntuples_cfg.py

```diff
@@ -768,7 +768,7 @@
 process.boostedJetUserDataAK8 = cms.EDProducer(
     'BoostedJetToolboxUserData',
     jetLabel  = cms.InputTag('jetUserDataAK8'),
-    puppiSDjetLabel = cms.InputTag('packedPatJetsAK8PFPuppiSoftDrop'),
+    puppiSDjetLabel = cms.InputTag('slimmedJetsAK8'),
```

Comment:
@alefisico 
The POG recipe quotes the MINIAOD collection, so I think they derived their corrections based on those values. I now checked the numbers I get in both collections and the difference is usually very small. I rather paste the results here.
My main worry, is that even some higher mass values are sometimes missing due to some subjets missing I think. I show the results for 100 events and show the largest differences in the ratio of uncorrected mass I got.
Based on this, I suggest to keep matching with the MINIAOD collection.

ratio     slimmed      packed
0.000000    8.470090    0.000000
0.000000    9.511510    0.000000
0.000000   62.090900    0.000000
0.000000   62.322200    0.000000
0.944192    6.984790    6.594980
0.991547    1.898820    1.882770
0.994754    1.519310    1.511340
0.994875    1.715200    1.706410
0.995489    1.961770    1.952920
0.997439    6.162610    6.146830
0.997977    3.109730    3.103440
0.998327    2.295940    2.292100
0.998630    6.227210    6.218680
0.998710    2.124630    2.121890
0.998751    3.570170    3.565710
0.999074   43.953600   43.912900
.... (~100 other jets)
1.000741    5.252100    5.255990
1.000981    3.994710    3.998630
1.001088    2.867510    2.870630
1.001294   62.671100   62.752200
1.001526    2.849850    2.854200
1.001556    3.489310    3.494740
1.001824    2.631770    2.636570
1.002926    3.253670    3.263190
1.002936    0.740472    0.742646
1.002955    3.847590    3.858960
1.004387   90.911200   91.310000
1.006554    2.284010    2.298980
1.007603   65.646300   66.145400
1.016990   17.951900   18.256900
1.158751    5.238330    6.069920